<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameCommandForm</class>
 <widget class="QWidget" name="FrameCommandForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame et al</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>22</y>
     <width>325</width>
     <height>227</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rbFrame">
      <property name="text">
       <string>Frame</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbRender">
      <property name="text">
       <string>Render</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbGrabSingle">
      <property name="text">
       <string>Grab - single</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbGrabOn">
      <property name="text">
       <string>Grab on</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbGrabOff">
      <property name="text">
       <string>Grab off</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbQuit">
      <property name="text">
       <string>Quit render</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rbRelease">
        <property name="text">
         <string>Release</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Handle:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leReleaseHandle"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QRadioButton" name="rbReset">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
